
<launch>
  <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
    <param name="autorepeat_rate" value="10" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="4" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="axis_boost" value="5" type="int"/>
  <param name="scale_linear" value="1" type="double"/>
  <param name="scale_angular" value="-1" type="double"/>
  <param name="minboost" value="0.15" type="double"/>
  <node pkg="joycontrol" type="teleop_joy" name="teleop"/>
</launch>
